features/protected_area.feature

```diff
@@ -0,0 +1,50 @@
+Feature: As a business owner
+  In order to update the menu and prevent others form doing it
+  I want to have a section that is protected
+
+Background:
+  Given there is a user with username "bob" and password "notbob"
+
+Scenario: Business owner should be able to access login form from login link
+  Given I visit the index page
+  And I click on the "Log In" link
+  Then I should see "Username:"
```

Comment:
You guys can delete this entire scenario, since you duplicate all of this in the other steps.